src/stories/components/Modal/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { ModalProps } from './types';
+
+import * as S from './styles';
+import { IconButton } from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+
+export const Modal = ({ children, closeModal, title, ...props }: ModalProps) => {
+  return (
+    <S.Wrapper>
+      <S.Dialog {...props} maxWidth={false} onClose={closeModal}>
+        <S.DialogTitle>
+          {title}
+          <IconButton
+            aria-label="close"
+            onClick={closeModal}
+            sx={{
```

Comment:
Sugiro colocar a estilização no arquivo `styles.ts` para manter o padrão de sempre que for necessário estilizar algum componente devemos sempre mexer no arquivo de estilos.